rancher/charts
# Include optional dashboards for logging stack

## Issue: 
[#46591](https://github.com/rancher/rancher/issues/46591)

## Problem
Implementing two optional Grafana dashboards to monitor the logging stack. Depends on [this already merged PR](https://github.com/rancher/charts/pull/4354).

## Testing
To test this feature a few steps are needed:
- First, it's necessary to enable the metrics.serviceMonitor option in the logging stack for both fluentd and fluentbit as implemented in the mentioned PR.
- Second, enable the loggingMonitors.fluentd and loggingMonitors.fluentbit options as implemented in this PR.
- You should now see two new dashboards in Grafana: `Rancher / Fluentd` and `Rancher / Fluentbit`.

## Engineering Testing
### Manual Testing


### Automated Testing


## QA Testing Considerations


### Regressions Considerations


## Backporting considerations
